CombineAIDriver.lua

```diff
@@ -1341,8 +1341,15 @@ function CombineAIDriver:startSelfUnload()
 		self.courseAfterPathfinding = nil
 		self.waypointIxAfterPathfinding = nil
 		local targetNode = fillRootNode or bestTrailer.rootNode
+		local trailerRootNode = bestTrailer.rootNode
+		local trailerLength = bestTrailer.sizeLength
+		self:debug('Trailer Length : %d', trailerLength)
+
+		local dX, dY, dZ = localToLocal(trailerRootNode, targetNode, 0, 0, 0)
```

Comment:
you can just do local _, _, dz = ... if you don't need dx and dy.

Reply:
Done in latest commit